### ISSUE TITLE
[DOCUMENTATION] Install instructions

### ISSUE THREAD

**Issue by spenserblack** [spenserblack/one-d-six-rs#12]
Note that `cargo install --git https://github.com/spenserblack/one-d-six-rs` can be used to install the version currently on github.

**Comment by spenserblack** [comment]
I probably won't add this to the README for now, since
- I hesitate to add more than basic usage to the README, to keep it from getting too wordy as this project ages
- It's probably safe to assume anyone who would want to use a version that is *not* released on Crates.io would know what they're doing, and would probably know how to install a crate from Github

Of course, I'm always willing to change my mind :slightly_smiling_face:

**Comment by Monadic-Cat** [comment]
You're probably right. It's best not to encourage installing unreleased versions.

**Comment by spenserblack** [comment]
Thanks! Closing issue for now.
